### ISSUE TITLE
Add Cypress.browser to the driver

### ISSUE THREAD

**Issue by brian-mann** [cypress-io/cypress#1919]
We should yield you the spec because it may be useful in the driver.

```bash
cypress run --browser chrome
```

```js
const browser = Cypress.browser

// return
{
  name: 'chrome',
  displayName: 'Chrome',
  version: '67.123.456.90',
  majorVersion: '67',
  path: '/path/to/browser'
}
```

**Comment by brian-mann** [comment]
@bahmutov also need your help with these typings too

**Comment by brian-mann** [comment]
Released in [`3.0.2`](https://docs.cypress.io/guides/references/changelog.html#3-0-2).
